### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#484] [Esami & Esoneri M-Z]
### Traccia

<img width="574" alt="image" src="https://github.com/user-attachments/assets/0385153f-9947-48a2-83b3-a34217c14a42">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/fb67436b-1849-4c45-bea2-2bda89a3596f)
